### ISSUE TITLE
It should not be possible to validate an object without category

### ISSUE THREAD

**Issue by grololo06** [ecotaxa/ecotaxa_dev#712]
It's a nonsense, but for now, it is possible. 

The category appears as "Unknown" in object detail, and shows as nothing under the image in classification page.

**Comment by grololo06** [comment]
Cleanup nightly code:
```
def RefreshAllProjectsStat():
    # Tout les objets validés sans classifications sont repassés en non validés
    ExecSQL("update obj_head oh set classif_qual=NULL where classif_qual='V' and classif_id is null ")

```

**Comment by grololo06** [comment]
The "cleanup" code has been removed, so it would be a good idea to prevent occurence soon.

**Comment by grololo06** [comment]
Fixed in 2.5.12
